beets/dbcore/queryparse.py

```diff
@@ -24,6 +24,10 @@
 PARSE_QUERY_PART_REGEX = re.compile(
     # Non-capturing optional segment for the keyword.
     r'(?:'
+    ur'(-|\u00ac)'  # Negation prefixes
+    r')?'
```

Comment:
For this part of the regex, we probably don't need the enclosing `(?: ... )`. Just `(-|\uXXXX)?` will suffice, unless I'm missing something subtle.